client/src/components/AddGuests.jsx

```diff
@@ -33,6 +39,8 @@ const AddGuests = ({ setTopLevelState, guests, deleteGuest }) => {
       <form onSubmit={onSubmit}>
         <label htmlFor='guest' > Add Guests </label>
         <input name='guest' type='text' value={name} onChange={onChange}/>
+        {duplicateName && <div>This guest is already present in your list.</div>}
+        {missingName && <div>Please enter a guest name.</div>}
```

Comment:
Changed the `div` to `p` for semantics and added a class for later styling.

```suggestion
        {nameError && <p class="error">This guest is already present in your list.</p>}
```